plugins/populator/frontend/public/javascripts/countly.models.js

```diff
@@ -107,12 +107,18 @@
 	}
     function getIAPEvents(){
         var iap = [];
-        var cur = countlyCommon.dot(countlyGlobal["apps"][countlyCommon.ACTIVE_APP_ID], 'plugins.revenue.iap_events');
-        if(cur && cur.length){
-            for(var i = 0; i < cur.length; i++){
-                if(cur[i] && cur[i].length){
-                    iap.push(cur[i]);
-                    eventsMap[cur[i]] = segments.Buy;
+        var cur = countlyGlobal["apps"][countlyCommon.ACTIVE_APP_ID].iap_event;
```

Comment:
iap event is now stored as plugin property, so should be obtained in a way that you have deleted on line 110